requirements.txt

```diff
@@ -1,31 +1,27 @@
-gunicorn
-
-Django>=3.2,<3.3
-django-webpack-loader==1.5.0
-mysqlclient
-django-mysql
-
-django-db-file-storage # Support for storage in the database
-Pillow # Needed for images
```

Comment:
@jonespm, I removed `Pillow`, since things seemed to work without it. Was this one of the libraries you were thinking about that was unused? Others? Most everything else seemed needed.

Reply:
Nevermind, seeing errors now, I'll add back `Pillow`. Must have seen some caching behavior.

Reply:
Yeah pillow is needed for `models.ImageField`.